cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceMeta.java

```diff
@@ -102,6 +102,10 @@ public NamespaceMeta build() {
     }
   }
 
+  public Id.Namespace getNamesapceId() {
```

Comment:
oh ok. Here is the typo :-) I thought it already existed. Can we change it to return ``NamespaceId`` too, instead?